.github/mergify.yml

```diff
@@ -77,6 +77,8 @@ pull_request_rules:
     conditions:
       - "updated-at<15 days ago"
       - "-draft"
+      - "-closed"
+      - "-merged"
```

Comment:
this leaves only draft.

Reply:
@obnoxxx Can you please explain if there is any real concern here?